tools/accuracy_checker/accuracy_checker/data_readers/data_reader.py

```diff
@@ -502,6 +502,24 @@ def read_item(self, data_id):
         return DataRepresentation(data, identifier=identifier)
 
 
+class NumpyBinReader(BaseReader):
+    __provider__ = 'numpy_bin_reader'
+
+    @classmethod
+    def parameters(cls):
+        params = super().parameters()
+        params.update({
+            "dtype": StringField(optional=True, default='float32')
```

Comment:
```suggestion
            "dtype": StringField(optional=True, default='float32', description='data type of returned data')
```

Reply:
not quiet correct description, because it is not only returned data type, it is also used for binary data parsing